CMakeLists.txt

```diff
@@ -444,19 +446,78 @@ elseif(UNIX)
   list(APPEND mxnet_LINKER_LIBS Threads::Threads)
 endif()
 
+if(BLAS STREQUAL "Open" OR BLAS STREQUAL "open")
```

Comment:
Can we keep this configuration in the ChooseBlas file? You can use PARENT_SCOPE option to modify the cmakelists scope variables: https://cmake.org/cmake/help/latest/command/set.html